server/routes/support/index.ts

```diff
@@ -11,7 +11,9 @@ import validationMiddleware from '../../middleware/validationMiddleware'
 import prisonIdCurrent from './types/prisonIdCurrent'
 import prisonIdAndEmail from './types/prisonIdAndEmail'
 import prisonIdDelete from './types/prisonIdDelete'
+import dateRange from './types/dateRange'
```

Comment:
This is all a bit weird.
These are references to the classes (types), so should all be upper cased.

Reply:
ok